alerting/src/main/kotlin/org/opensearch/alerting/settings/AlertingSettings.kt

```diff
@@ -21,6 +21,7 @@ class AlertingSettings {
         const val DEFAULT_PERCOLATE_QUERY_NUM_DOCS_IN_MEMORY = 50000
         const val DEFAULT_PERCOLATE_QUERY_DOCS_SIZE_MEMORY_PERCENTAGE_LIMIT = 10
         const val DEFAULT_DOC_LEVEL_MONITOR_SHARD_FETCH_SIZE = 10000
+        const val DEFAULT_FAN_OUT_NODES = 1000
```

Comment:
Minor: 1000 seems arbitrary. If we are trying to avoid limiting the max number of fan out nodes by default, this could be `Int.MAX_VALUE` for clarity